@nikhilwoodruff Thanks for this PR!

I'm seeing test failures on GH Actions and am getting them locally as well.  I changed all references to 2018 to 2019 in `test_get_micro_data.py` as it seemed that was raising errors in that no 2018 file seemed available to download.  Doing so fixed one error, but this remains:

```
(oguk-dev) jason.debacker@JDEBACKER-2 oguk % pytest tests/test_get_micro_data.py
/Users/jason.debacker/anaconda3/envs/oguk-dev/lib/python3.7/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
=========================================================== test session starts ============================================================
platform darwin -- Python 3.7.10, pytest-5.4.3, py-1.11.0, pluggy-0.13.1
rootdir: /Users/jason.debacker/repos/OG-UK
plugins: xdist-2.5.0, anyio-3.5.0, forked-1.4.0, pep8-1.0.6
collected 3 items

tests/test_get_micro_data.py F..                                                                                                     [100%]

================================================================= FAILURES =================================================================
_________________________________________________________________ test_frs _________________________________________________________________

    def test_frs():
        """
        Check that setting `data` to 'frs' uses cps data
        """
        baseline = False
        start_year = 2019

        # create a parametric reform
        def lower_pa(parameters):
            parameters.tax.income_tax.allowances.personal_allowance.amount.update(
                period=2019, value=10000
            )
            return parameters

        class lower_personal_tax_allowance(Reform):
            def apply(self):
                self.modify_parameters(modifier_function=lower_pa)

        reform = lower_personal_tax_allowance

        calc_out = get_micro_data.get_calculator_output(
>           baseline, start_year, reform=reform, data="frs"
        )

tests/test_get_micro_data.py:47:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
get_micro_data.py:87: in get_calculator_output
    sim = Microsimulation(reform, **sim_kwargs)
../../../anaconda3/envs/oguk-dev/lib/python3.7/site-packages/openfisca_uk/tools/simulation.py:55: in __init__
    f"\nYear {year} not available in dataset {dataset.name}: \n\t* Download the dataset [y]\n\t* Use the synthetic FRS (and set default) [n]\n\nChoice: "
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <_pytest.capture.DontReadFromInput object at 0x7faca85b9e10>, args = ()

    def read(self, *args):
        raise IOError(
>           "pytest: reading from stdin while output is captured!  Consider using `-s`."
        )
E       OSError: pytest: reading from stdin while output is captured!  Consider using `-s`.

../../../anaconda3/envs/oguk-dev/lib/python3.7/site-packages/_pytest/capture.py:733: OSError
----------------------------------------------------------- Captured stdout call -----------------------------------------------------------

Year 2018 not available in dataset synth_frs:
	* Download the dataset [y]
	* Use the synthetic FRS (and set default) [n]

Choice:

========================================================= short test summary info ==========================================================
FAILED tests/test_get_micro_data.py::test_frs - OSError: pytest: reading from stdin while output is captured!  Consider using `-s`.
================================================= 1 failed, 2 passed, 6 warnings in 9.89s ==================================================
(oguk-dev) jason.debacker@JDEBACKER-2 oguk % pytest tests/test_run_example.py
```

I'm unsure what the traceback in telling me since the start year is 2019, so I don't know where the warning about 2018 not being available is coming from.  What do you think?

--- a/.github/publish-git-tag.sh
+++ b/.github/publish-git-tag.sh
@@ -1,4 +1,4 @@
 #! /usr/bin/env bash
 
 git tag `python setup.py --version`
-git push --tags  # update the repository version
+git push --tags || true  # update the repository version

--- a/.github/workflows/deploy_docs.yml
+++ b/.github/workflows/deploy_docs.yml
@@ -36,3 +36,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           BRANCH: gh-pages # The branch the action should deploy to.
           FOLDER: docs/book/_build/html # The folder the action should deploy.
+      
+      - name: Publish a git tag
+        run: ".github/publish-git-tag.sh"

--- a/examples/plot.py
+++ b/examples/plot.py
@@ -6,7 +6,7 @@ from argparse import ArgumentParser
 import plotly.express as px
 
 
-def main(year: int = 2018):
+def main(year: int = 2019):
     year = str(year)
     # read in tax function parameters from pickle
     tax_funcs_base = safe_read_pickle(
@@ -148,6 +148,6 @@ def get_tax_fn(
 
 if __name__ == "__main__":
     parser = ArgumentParser()
-    parser.add_argument("--year", type=int, default=2018)
+    parser.add_argument("--year", type=int, default=2019)
     args = parser.parse_args()
     main(args.year)

--- a/examples/run_oguk.py
+++ b/examples/run_oguk.py
@@ -21,13 +21,13 @@ def get_default_reform():
     from openfisca_tools.reforms import set_parameter
 
     return set_parameter(
-        "tax.income_tax.rates.uk[0].rate", 0.3, "year:2018:10"
+        "tax.income_tax.rates.uk[0].rate", 0.3, "year:2019:10"
     )
 
 
 start_time = time.time()
 # Set start year and last year
-START_YEAR = 2018
+START_YEAR = 2019
 from ogcore.parameters import Specifications
 
 

--- a/oguk/get_micro_data.py
+++ b/oguk/get_micro_data.py
@@ -79,7 +79,7 @@ def get_calculator_output(baseline, year, reform=None, data=None):
 
     """
     # create a simulation
-    sim_kwargs = dict(dataset=dataset, year=2018)
+    sim_kwargs = dict(dataset=dataset, year=2019)
     if reform is None:
         sim = Microsimulation(**sim_kwargs)
         reform = ()

--- a/oguk/tests/test_get_micro_data.py
+++ b/oguk/tests/test_get_micro_data.py
@@ -28,7 +28,7 @@ def test_frs():
     Check that setting `data` to 'frs' uses cps data
     """
     baseline = False
-    start_year = 2018
+    start_year = 2019
 
     # create a parametric reform
     def lower_pa(parameters):
@@ -62,9 +62,9 @@ def test_household_mtr_calculation():
     mtr_x = get_micro_data.get_household_mtrs(
         (),
         "employment_income",
-        2018,
+        2019,
         dataset=get_micro_data.dataset,
-        year=2018,
+        year=2019,
     )
     assert mtr_x.isna().sum() == 0
     assert mtr_x.min() >= 0
@@ -73,9 +73,9 @@ def test_household_mtr_calculation():
     mtr_y = get_micro_data.get_household_mtrs(
         (),
         "savings_interest_income",
-        2018,
+        2019,
         dataset=get_micro_data.dataset,
-        year=2018,
+        year=2019,
     )
     assert mtr_y.isna().sum() == 0
     assert mtr_y.min() >= 0

